biosynseq/generate.py

```diff
@@ -0,0 +1,122 @@
+"""Generate synthetic sequences and embeddings given model weights."""
+import logging
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+
+import numpy as np
+import pytorch_lightning as pl
+import torch
+from gene_transformer.config import ModelSettings
+from gene_transformer.model import (
+    LoadDeepSpeedStrategy,
+    LoadPTCheckpointStrategy,
+    ModelLoadStrategy,
+    inference,
+)
+from gene_transformer.utils import non_redundant_generation, seqs_to_fasta
+
+logger = logging.getLogger(__name__)
+
+
+def generate_fasta(
+    model_strategy: ModelLoadStrategy, fasta_path: str, num_seqs: int
+) -> dict:
+    """Given pt or deepspeed file, output generated sequences' fasta files."""
+    # obtain model
```

Comment:
See if you can use the docstring generator and fill out the numpy-style docstring comments: https://numpydoc.readthedocs.io/en/latest/format.html